code/modules/mining/machine_processing.dm

```diff
@@ -103,6 +103,16 @@
 		else
 			machine.selected_clown = 0
 
+		//titanium
+		if(machine.ore_titanium)
+			if (machine.selected_titanium==1)
+				dat += text("<A href='?src=\ref[src];sel_titanium=no'><font color='green'>Smelting</font></A> ")
+			else
+				dat += text("<A href='?src=\ref[src];sel_titanium=yes'><font color='red'>Not smelting</font></A> ")
+			dat += text("Titanium: [machine.ore_titanium]<br>")

```

Comment:
these ```text()``` calls aren't actually doing anything, if the ```text()``` calls in this code have no second arg
(```text(first_arg , second_arg)```) then they should be remove (leaving behind just the ```dat += "your text"```)